client/extensions/talk-to-plugin/query-plugin.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { isRequestingSiteOptions } from './state/selectors';
+import { fetchOptions } from './state/actions';
+
+class QueryPlugin extends Component {
+	static propTypes = {
+		siteId: PropTypes.number.isRequired,
+		requestingOptions: PropTypes.bool,
```

Comment:
I would name it `isRequestingOptions` to minimize confusion and for consistency with Calypso tendencies.